templates/default/test/test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const func = require('..');
+const test = require('tape');
+const cloudevents = require('cloudevents-sdk/v1');
+
+// Ensure that the function completes cleanly when passed a valid event.
+test('Handles a valid event', t => {
+  // A valid event includes id, type and source at a minimum.
+  let e = cloudevents.event();
+  e.id('TEST-EVENT-01');
+  e.type('com.example.cloudevents.test');
+  e.source('http://localhost:8080/');
```

Comment:
Eventually, it might be nice to provide some level of framework testing support so that it's easier for a function developer to inject events. This isn't bad though.

Reply:
Sounds like a good idea to me.  I would like to see these tests grow substantially as they could be a good first-stop for users to see what the context is guaranteeing.  I would assume that after a few tests all having to create events, a helper will look quite attractive!